<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="usrLoginLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLoginLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="profPhoto">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../graphSource/profPhoto1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>421</width>
     <height>291</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>421</width>
     <height>291</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../graphSource/loginBackground1.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="regBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPwdBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <zorder>backGround</zorder>
  <zorder>usrLoginLineEdit</zorder>
  <zorder>pwdLoginLineEdit</zorder>
  <zorder>loginBtn</zorder>
  <zorder>profPhoto</zorder>
  <zorder>regBtn</zorder>
  <zorder>findPwdBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
